### PR TITLE
create graphs for paper with netgraph package

### PR DESCRIPTION
**Pull request by jonashund**

### CODE REVIEW

**Comment by JulianKarlBauer** [comment]
Nice, thanks! We should definitely use your nice graphic inside the paper!

File: paper/paper.md [hunk 1]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
 
 The Python package `mechkit` is a toolkit for researchers
 in the field of continuum mechanics and material modeling.
-`Mechkit` contains methods and operators
-that simplify common tasks - especially in the area of tensor algebra and tensor notation.
+`mechkit` **Why capitalized?** contains methods and operators
+that facilitate common tasks within tensor algebra and tensor notation.
```
Comment (JulianKarlBauer) [inline paper/paper.md:36]:
```suggestion
`mechkit` contains methods and operators
that simplify common tasks - especially in the area of tensor algebra and tensor notation.
```

File: paper/paper.md [hunk 2]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
 
-The implementations in `mechkit` aim at usability, seek to provide understandable source code
+The implementations in `mechkit` aim at usability, seek to provide understandable source code,
 and do not put major emphasis on performance.
 Furthermore, the implementations follow, as directly as possible,
 the notation and formulation in the respective scientific sources.
 A redundant implementation of identical operations based on different
-sources is strived for.
-
+sources is strived for. **For validation or just for fun?**
```
Comment (JulianKarlBauer) [inline paper/paper.md:55]:
```suggestion
sources is aimed at, for validation reasons.
```

File: paper/paper.md [hunk 3]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
-The methods and operators occurring in the field of linear elasticity and material modeling are
-comparatively simple and compact.
-Due to this simplicity, they are usually implemented independently by each scientist
-during the studies and
-do not find their way into more general libraries which are common in other research fields such as physics or computer science.
-Apart from didactic advantages, this procedure causes problems for
+The methods and operators for the description of linear elasticity and material modeling are mathematically simple and can be expressed in a compact manner.
+Due to this relative simplicity, they are usually implemented independently by each scientist for use in their studies.
+Consequentially, unlike in other research field such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.
+Apart from didactic advantages **I don't get what you want to say here. Suggestion in line 65.**, this procedure causes problems for
```
Comment (JulianKarlBauer) [inline paper/paper.md:62]: Thanks for feedback, let's talk about this, but please remove your changes.
Why are all these lines highlighted, although you did just add a comment?
Please do not change line endings.

File: paper/paper.md [hunk 4]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
 the exchange of research code and
 the reliability of the implementations.
+This presents a major obastacle with regard to the exchange and reliability of research code as well as the interpretation and comparison of results.
```
Comment (JulianKarlBauer) [inline paper/paper.md:65]: nice, thanks

File: paper/paper.md [hunk 5]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
 
-The main goal of the project `mechkit` is the reuse of research code,
-to increase the reliability of the research results and
-accelerate and simplify further research.
-`Mechkit` is inspired by [@fiberoripy] and the projects of the author of [@meshio].
+The main goal of the project `mechkit` is reusable research code that increases the reliability of the research results and
+accelerate as well as simplifies further research.
+`mechkit` is inspired by [@fiberoripy] and the projects of the author of [@meshio], **e.g.**.
```
Comment (JulianKarlBauer) [inline paper/paper.md:70]: Please avoid no-content-changes and remove comments here, you would change the authorship...

File: paper/paper.md [hunk 6]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
 
 ## Motivation by example: Isotropic material and notations
 
 In the overlapping area of theoretical continuum mechanics, experimental
-material modeling
-and the numerical solution methods for boundary value problems
+material modeling,
+and numerical solution methods for boundary value problems,
```
Comment (JulianKarlBauer) [inline paper/paper.md:75]: nice

File: paper/paper.md [hunk 7]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
 a multitude of different notations exist.
-As an example, consider the description of the mechanical properties of a
+As an example, one may consider the description of the mechanical properties of a
```
Comment (JulianKarlBauer) [inline paper/paper.md:77]: thanks

File: paper/paper.md [hunk 8]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
 homogeneous and isotropic, i.e. direction-independent, material within the framework of linear elasticity.
 Such a material can be described identically by two scalar material parameters.
-However, in the disciplines mentioned above, at least six different material parameters are commonly used
-and motivated by different applications and measurement methods.
+However, in the disciplines mentioned above, at least six different material parameters are commonly used,
+motivated by different applications and measurement methods.
 This results in fifteen possible combinations of scalar descriptions of an
-isotropic material, which can be combined to a fourth-order elasticity tensor.
```
Comment (JulianKarlBauer) [inline paper/paper.md:86]: I would like to stick to `a`

File: paper/paper.md [hunk 9]
```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
...
-Since the necessary number of translation functions between different notations
-increases drastically with increasing number of notations,
-translation between all notations might not be practical.
-For the case of notations of second- and fourth-order tensors,
-the shortest path between source and target notation is determined by
+The translation between different notations is often tedious and prone to errors.
+`mechkit` allows an easy exchange between different notations with user-friendly
+interfaces, thereby preventing errors.
+The necessary number of translation functions between different notations
+increases drastically **exponentially?** with an increasing number of notations.
```
Comment (JulianKarlBauer) [inline paper/paper.md:91]:
```suggestion
increases drastically with an increasing number of notations.
```

**Review by JulianKarlBauer — CHANGES_REQUESTED** [review]

**Comment by JulianKarlBauer** [comment]
Hi Jonas, thanks for adding this valuable contribution.
Would you mind working on these few requested changes?

A general remark:
Please do not include comments into the source.
You can use githubs comment features within merge-request, see, e.g., [help](https://docs.gitlab.com/ee/user/project/merge_requests/reviews/)

**Comment by jonashund** [comment]
Hi Julian,

I gave the review tutorial a quick read and will try the function later.
I'd also like to appear in the authors list.
Is my merge request now clean and ready to merge?

**Review by JulianKarlBauer — APPROVED** [review]

**Pull request merged**